README.md

```diff
@@ -13,4 +13,5 @@ Docker Tutorials project contains Session PPT's & Lab sessions of Docker Series
   
 ## Lab sessions
   1. [Building docker image from running container](lab_sessions/build_docker_image_from_a_running_container.md)
-  2. [Wordpress setup with multiple containers](lab_sessions/Deploy_Wordpress_With_Multiple_Container.md)
\ No newline at end of file
+  2. [Wordpress setup with multiple containers](lab_sessions/Deploy_Wordpress_With_Multiple_Container.md)
+  3. [Dockerize Shell Script Simple](lab_sessions/Dockerize_Shell_Script_Simple.md)
```

Comment:
Link text should be Dockerizing shell script

Reply:
actually sir i am going to add one more lab for  Dockerizing shell script